src/modules/news/news.js

```diff
@@ -1,126 +1,168 @@
+import { NewsApi } from './newsApi';
+import { Notification } from '../../shared/notifications/notifications';
+import { createElement } from '../../utils/createElement';
+
 export class News {
     constructor() {
         this.viewElement = document.querySelector('#main');
+        this._api = new NewsApi();
+        this.start();
+    }
+
+    async start() {
+        try {
+            await this._fetchData();
+            this._createTitle();
+            this._createForm();
+        } catch (error) {
+            new Notification().showError('Fetch news data error', error);
+        }
+    }
+
+    async _fetchData() {
+        this._topNews = await this._api.getTopNews('top-headlines').then((response) => {
+            let main = this._createContainer();
+            let data = response.articles;
+            data.forEach((article) => {
+                console.log(article);
+                main.appendChild(this._createCard(article));
+            });
+        });
     }
 
-    start() {
-        this._createTitle();
-        this._createForm();
-        this._createContainer();
+    async _searchData(keyword) {
+        this._searchNews = await this._api.getSearchedNews(keyword).then((response) => {
```

Comment:
same thing 

are you sure this will return a value and assign it to the this._searchNews?